### DISCUSSION TITLE
Extend double point bug report bonus to all remaining TBC content yet to be released.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#7175] [Suggestions]
The earlier reports come out, the more time there is to fix them. I see no reason not to incentivize players going out of their way to check out unreleased content on the PTR. It's a good incentive for the next raid release, but in my opinion it'd be even better to extend to the rest of the PTR-only content.

**Comment by Nyeriah** [comment]
It has been accepted.
